source/js/components/petition/petition.jsx

```diff
@@ -116,26 +117,24 @@ export default class Petition extends React.Component {
     );
   }
 
-  fbButtonClicked() {
-    this.shareButtonClicked(`#share-progress-fb`);
-  }
-
-  twButtonClicked() {
-    this.shareButtonClicked(`#share-progress-tw`);
-  }
-
-  emButtonClicked() {
-    this.shareButtonClicked(`#share-progress-em`);
-  }
-
-  shareButtonClicked(id) {
+  shareButtonClicked(event, shareProgressButtonId) {
```

Comment:
Where's `shareProgressButtonId` coming from?

Reply:
e.g.,

```html
<button
  className="btn btn-secondary btn-share facebook-share"
  onClick={e => this.shareButtonClicked(e, `share-progress-fb`)}
>
```

`shareProgressButtonId` is `share-progress-fb` in this case

Reply:
not sure why I missed that lol but cool!